src/main/java/com/csdemo/coffeestore/service/OrderService.java

```diff
@@ -25,59 +25,15 @@
 	private CartRepository  cartRepo;
 	@Autowired
 	private OrderRepository ordersRepo;
-
-//    public List<CartResponse> placeOrder(List<CartRequest> cartRequests) {
-//        this.cartRequests = cartRequests;
-//        List<CartResponse> ResponceList = new ArrayList<>();
-//        CartResponse responseObj;
-//        int totalPrice = 0;
-//        String Message = "item not available";
-//
-//        for (CartRequest cartRequest : cartRequests) {
-//            responseObj = new CartResponse();
-//            Items repoItem = itemsRepo.findByname(cartRequest.getName());
-//
-//            if (repoItem != null) {
-//                BeanUtils.copyProperties(repoItem, responseObj);
-//                int orderQuantity = cartRequest.getQuantity();
-//                int Iteamprice = responseObj.getPrice();
-//
-//                int actual = orderQuantity;
-//                int repoQuantity = repoItem.getQuantity();
-//                if (repoQuantity > 0) {
-//                    if (repoQuantity < orderQuantity) {
-//                        actual = Math.abs(repoQuantity - orderQuantity);
-//
-//                    }
-//                    Message = actual + "/" + orderQuantity + " Avaiable";
-//                    totalPrice = actual * Iteamprice;
-//                }
-//            } else {
-//                Message = "Item Not available in the Menu";
-//                responseObj.setName(cartRequest.getName());
-//            }
-//
-//            responseObj.setPrice(totalPrice);
-//            responseObj.setAvailability(Message);
-//            ResponceList.add(responseObj);
-//
-//        }
-//
-//        return ResponceList;
-//    }
 	
 	public OrderConfirmation createOrderResponse ( int OrderID ) {
-		List < Cart > cartItems=cartRepo.findByCartId ( OrderID );
-		
-		OrderConfirmation confirmation=new OrderConfirmation ( );
-		
+		List < Cart >         cartItems   =cartRepo.findByCartId ( OrderID );
+		OrderConfirmation     confirmation=new OrderConfirmation ( );
 		List < CartResponse > responceList=confirmation.getResponceList ( );
 		CartResponse          responseObj;
 		int                   orderTotal  =0;
 		
-		
 		for ( Cart item : cartItems ) {
-			
 			int    totalPrice=0;
 			String Message   ="item not available";
```

Comment:
indentation and instead of hardcoding the messages, u can probably store them in a constants file or create another property to store the message. Doing so ensures : common place of change if at all msg needs to be updated and easy to maintain and read